t/Test28-fregrid_stretched.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bats
+# test stretched grid 
+
+@test "Test stretched grid data lats 32.0 34.0 35.4" {
+
+  if [ ! -d "Test28" ] 
+  then
+  		mkdir Test28
+  fi
+
+  cd Test28
+   cp $top_srcdir/t/Test28-input/ocean_hgrid.nc . 
+   cp $top_srcdir/t/Test28-input/ocean_mosaic.nc .
+   cp $top_srcdir/t/Test28-input/topog.nc .
+
+#Make streetched grid 
+  run command make_hgrid \
+               --grid_type gnomonic_ed --do_schmidt \
+               --stretch_factor 2.5 \
+               --target_lon 262.4 \
+               --target_lat 32.0 \
+               --nlon 512 \
+               --grid_name C384_grid_32.0
```

Comment:
These are actually C256 (nlon/2) not C384.

Reply:
Thanks for that detail info, we can change that.